.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           python-version: "3.x"
       - name: Install conan
         run: |
-          pip install -U --no-cache-dir conan
+          pip install -U --no-cache-dir conan==1.59
```

Comment:
Not sure if this and further (conan) changes below are related to this feature. If not, for the sake of separation of concerns shall be (whenever possible) extracted into a different PR allowing the changes (if needed) to be integrated as soon as possible into the master -> so other contributions could benefit of it

Reply:
Good point. It's the time issue again.